custom_components/frigate/__init__.py

```diff
@@ -196,6 +202,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     client = FrigateApiClient(
         str(entry.data.get(CONF_URL)),
         async_get_clientsession(hass),
+        str(entry.data.get(CONF_USERNAME)),
```

Comment:
As those parameters will not be present for anyone today, `entry.data.get(CONF_USERNAME)` will return `None` and then you'll cast that into a string resulting it the string `'None'`. This is probably wrong? It would be better to just use `None` as meaning no username or password (rather than `""`).